templates/lib/before_deploy.sh

```diff
@@ -0,0 +1,32 @@
+# This script takes care of building the crate and packaging it for release.
+
+PKG_NAME="{{PKG_NAME}}"
+
+set -ex
+
+main() {
+  local src=$(pwd) \
+    stage=
+
+  case $TRAVIS_OS_NAME in
+    linux)
+      stage=$(mktemp -d)
+      ;;
+    osx)
+      stage=$(mktemp -d -t tmp)
+      ;;
+  esac
+
+  test -f Cargo.lock || cargo generate-lockfile
+
+  cross rustc --bin $PKG_NAME --target $TARGET --release -- -C lto
+  cp target/$TARGET/release/$PKG_NAME $stage/
```

Comment:
I think cargo generates libs in the form of `${PKG_NAME}lib.so`

Reply:
I thought it was `lib${PKG_NAME}.so` unless you manually set the `lib.name` field?

Reply:
Oh, you are right!